apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.html.ts

```diff
@@ -8,21 +8,39 @@ const examplePrompts = [
     id: 'random_mc',
     question:
       'Write a multiple choice question asking the user to choose the median of 5 random numbers between 1 and 100. Display all of the numbers to the user, and then make each of the numbers a potential answer, with the median being the correct answer.',
+    userPrompt:
+      'Each random number generated should be a potential answer to the multiple-choice question. Randomize the order of the numbers.',
+    prompt:
+      'Write a multiple choice question asking the user to choose the median of 5 random numbers between 1 and 100. Display all numbers to the user, and ask them to choose the median.',
+    gradingPrompt: 'Make the actual median the correct answer.',
```

Comment:
- Write these in the order that they'll actually be used (`prompt`, `userPrompt`, `gradingPrompt`).
- Please please please use consistent naming. Here these are `prompt`, `userPrompt`, and `gradingPrompt`; elsewhere they are called `promptGeneral`, `promptUserInput`, and `promptGrading`. I don't have a strong preference for exactly which one you use, please just be consistent.

Reply:
Changed in 09ebd145316f730bdcbac5525887fda484b63483